driver/ppm_fillers.c

```diff
@@ -5715,7 +5715,8 @@ int f_sys_dup_e(struct event_filler_arguments *args)
 	 * oldfd
 	 */
 	syscall_get_arguments_deprecated(args, 0, 1, &val);
-	res = val_to_ring(args, val, 0, false, 0);
+	int32_t fd = (int32_t) val;
```

Comment:
int32_t fd = 0 should be declared at the beginning of the function we cannot mix declaration with the code, ISO C90 forbids mixed declarations and code

Reply:
same in all other places

Reply:
*facepalm*